release/lts/nanoserver20H2/dependabot/Dockerfile

```diff
@@ -3,4 +3,4 @@
 
 # Dummy docker image to trigger dependabot PRs
 
-FROM mcr.microsoft.com/windows/nanoserver:2004
```

Comment:
I feel like nano 2k4 should be kept for LTS, and only introduce this new version in stable and preview channels, since it is new.  Even if the LTS supports it, it seems like its a bit late in the LTS lifespan to introduce a new OS version now.

Reply:
This change is fine, as long as 2004 image is not deleted